docs/README.md

```diff
@@ -28,6 +28,11 @@ Everything about the basic & default `urql` behaviour
 that comes with explanations of the `fetchExchange`,
 the `cacheExchange`, and how to use _Subscriptions_.
 
+### [Exchanges](exchanges.md)
+
+Exchanges written by the community, these enhance
+the behavior of your urql-client.
```

Comment:
We should probably stuck to writing urql here as just urql in an inline code annotation and without client?

And maybe we can reuse this quote from the announcement blog post:

> customize and augment every aspect of the GraphQL client from how data is cached to how components receive their data

Also I think this list won't be just consistent of community exchanges but also "official" ones